### DISCUSSION TITLE
Can I change the background color of the bar?

### DISCUSSION THREAD

**Discussion by alexbkang** [noib3/cokeline.nvim#110] [General]
<img width="1010" alt="Screen Shot 2023-05-21 at 1 34 07 AM" src="https://github.com/willothy/nvim-cokeline/assets/63813437/0331cb48-d56c-4782-8707-dacf702a306f">

Theres a background color (dark gray/black) that gets set in the barline. As far as I can tell (in the docs), there is not a setting for it.

Is there a way I could change the color?

I am using the Kanagawa theme if that matters.

Thanks so much.

**Comment by willothy** [comment]
The `Tabline` highlight group should work.
